upgrading/topics/keycloak/changes-18_0_0.adoc

```diff
@@ -0,0 +1,20 @@
+= Step-up authentication
+
+Step-up authentication is new feature added. There is `acr` client scope, which contains protocol mapper, which is supposed to add `acr`
```

Comment:
It might be good that L3-L5 be one sentence.

Reply:
I've kept 2 sentences, but I've changed the second sentence to start with `The` instead of `So` .

Reply:
Sorry. What I wanted to say is to remove linefeeds among L3-L5. Not write as them one sentence.

Reply:
Replied in the comment https://github.com/keycloak/keycloak-documentation/pull/1428#discussion_r824468705 . We can keep the conversation there related to line feeds.

Reply:
Yes, I agree.